docs/fr/week12/12-3.md

```diff
@@ -49,7 +49,7 @@ $$
 \boldsymbol{h} = \boldsymbol{X} \boldsymbol{a}
 $$
 
-where $\boldsymbol{a} \in \mathbb{R}^n$ is a column vector with components $\alpha_i$.
+where $\boldsymbol{a} \in \mathbb{R}^t$ is a column vector with components $\alpha_i$.
```

Comment:
This is the comment (which you correctly fixed), but you forgot to edit line 81 below.